jenkins/Step_by_Step.md

```diff
@@ -0,0 +1,88 @@
+# How to create a new user for cloud-governance
+1. Create a IAM policy CloudGovernanceDeletePolicy
+   1. Use [CloudGovernanceDeletePolicy.json](iam/clouds/aws/CloudGovernanceDeletePolicy.json) to create the policy 
+2. Create **cloud-governance-user** and add the above created policy.
+3. Create s3 bucket to store policy results.
+
+# Adding jenkins slave
+1. Install java-11-jdk 
+    ```commandline
+    sudo yum install java-11-openjdk-devel
+    ```
+2. Install docker on [Fedora](https://docs.docker.com/engine/install/fedora/)
+   ```commandline
+    sudo dnf -y install dnf-plugins-core
+    sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+    sudo dnf install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin docker-compose
+    sudo systemctl start docker   
+    ```
+3. Create Jenkins user
+   ```commandline
+    useradd jenkins -U -s /bin/bash
+   passwd jenkins
+    ```
+4. Add jenkins user to sudoers file
+    ```
+   $ vi /etc/sudoers
+   jenkins ALL=(ALL) NOPASSWD: ALL
+   ```
+5. Giving permissions to jenkins user to run docker container
+    ```
+   sudo chown jenkins:jenkins /var/run/docker.sock
+    ```
+
+6. Run the cloud_governance_stack [ ElasticSearch, Kibana, Grafana]
+    ```commandline
+    # detached mode
+    docker-compose -f docker_compose_file_path up -d
```

Comment:
pls put the link to d[ocker-compose.yml](https://github.com/redhat-performance/cloud-governance/pull/602/files#diff-0b3bf01515c785027e95f6dcb6cbec90e305189dc3c25a0c522d4070eaa743a4)